src/libraries/System.Runtime.Serialization.Xml/ref/System.Runtime.Serialization.Xml.cs

```diff
@@ -75,6 +119,41 @@ public partial class DataContractSerializerSettings
         public System.Xml.XmlDictionaryString? RootNamespace { get { throw null; } set { } }
         public bool SerializeReadOnlyTypes { get { throw null; } set { } }
     }
+    public sealed partial class DataContractSet
+    {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public void Add(Type type) { throw null; }
+        public static void CompileSchemaSet(System.Xml.Schema.XmlSchemaSet schemaSet) { throw null; }
+        public DataContractSet(ISerializationSurrogateProvider? dataContractSurrogate, System.Collections.Generic.ICollection<Type>? referencedTypes, System.Collections.Generic.ICollection<Type>? referencedCollectionTypes) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public DataContractSet(DataContractSet dataContractSet) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public void ExportSchemaSet(System.Xml.Schema.XmlSchemaSet schemaSet) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public DataContract GetDataContract(Type type) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public DataContract? GetDataContract(System.Xml.XmlQualifiedName key) { throw null; }
+        public System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<System.Xml.XmlQualifiedName, DataContract>> GetEnumerator() { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
```

Comment:
Did you forget to add an IEnumerator interface to the type definition?

Reply:
Nope. DataContractSet did not implement IEnumerable in 4.8. Just doing what 4.8 did.
But after our brief chat earlier, this has been kept internal so it doesn't cause problems with API reviews, and we expose the Contracts collection instead, which is how the schema importer can enumerate.